modules/460-log-shipper/hooks/testdata/file-to-kafka/manifests.yaml

```diff
@@ -20,6 +20,7 @@ spec:
     bootstrapServers:
     - "192.168.1.1:9200"
     topic: "logs"
+    keyField: host
```

Comment:
we need second case when this field is absent, because this field isn't mandatory

Reply:
this check is in [file](https://github.com/deckhouse/deckhouse/blob/89f753e5e8482cfecaa819581bd172296b45ffb1/modules/460-log-shipper/hooks/testdata/file-to-kafka-tls/manifests.yaml)